debian/autobake-deb.sh

```diff
@@ -105,36 +151,11 @@ case "${LSBNAME}" in
     fi
     ;&
   sid)
-    # should always be empty here.
-    # need to match here to avoid the default Error however
-    ;;
-    # UBUNTU
-  bionic)
-    remove_rocksdb_tools
-    [ "$architecture" != amd64 ] && disable_pmem
-    ;&
-  focal)
-    replace_uring_with_aio
-    ;&
-  impish|jammy|kinetic)
-    # mariadb-plugin-rocksdb s390x not supported by us (yet)
-    # ubuntu doesn't support mips64el yet, so keep this just
-    # in case something changes.
-    if [[ ! "$architecture" =~ amd64|arm64|ppc64el|s390x ]]
-    then
-      remove_rocksdb_tools
-    fi
-    if [[ ! "$architecture" =~ amd64|arm64|ppc64el ]]
-    then
-      disable_pmem
-    fi
-    if [[ ! "$architecture" =~ amd64|arm64|armhf|ppc64el|s390x ]]
-    then
-      replace_uring_with_aio
-    fi
+    # The default packaging should always target Debian Sid, so in this case
```

Comment:
Is there some specific reason to move these up? Just alphabetical order?

Reply:
Otto moving these around doesn't actually have a functional purpose and makes merging harder.

Reply:
It makes more sense to have all Ubuntu first, then all Debian and finally at the end the fallback Sid (instead of having the fallback in the middle of the list).